### DISCUSSION TITLE
769

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#938] [Comments]
# 769

주말에 쉬는 중 다음과 같은 링크를 공유 받았다. 시니어만 고용하는 것은 소프트웨어 업계에서 최악의 정책입니다 글을 읽어보면 시니어를 뽑지말아야한다는 것은 아니지만, 시니어만으로 팀을 꾸릴려고 해서는 안된다는 이야기를 한다. 대체적으로 내용에 꽤 공감을 했다. 특히 "ambition, character and brains have little to do with experience." (야망, 성격, 두뇌는 경험과는 거의 관련이 없습니다.) 는 매우 공감이 갔다. 글을 읽다가 소라의 날개 10권에서 주인공 팀과 겨룬 신죠 토와 고교 농구부가 생각이 많이 났다. 신죠 토와 고교 농구부는 다카하시와 코지마 2명의 학생이 입학을 하면서 농구부가 없던 학교에 농구부를 신설하고, 부원들을 모집하면서 시작한다. 하..

https://jojoldu.tistory.com/769

**Comment by soulduse** [comment]
형이 자주 사용하시던 짤이 저 내용이었군요 ㅎㅎ
좋은글 감사합니다. 
신입 채용이 무조건적인 비용이라고만 생각했던적이 있었는데, 같이 일하다보니 잘하시는 분들은 금방 1인분을 해내시는것을 보고 
면접과 채용이 정말 중요하다고 느끼기도 하고, 어느순간 부정적인 느낌이 많이 사라지더라구요 ㅎㅎ 
사실 회사일은 늘 전쟁터와 같아서 쉴틈없고, 저 같은 경우 거의 개발할 시간없이 회의만 하루종일 하다보면 하루가 끝나곤 했는데 그런 와중에서도 옆에서 잘 케어(?) 해줄 시니어 팀원이 있다면 정말 든든함이 되더라구요. 많은 생각을 하게 되는 글 감사합니다!

**Comment by SpringDream0406** [comment]
좋은 글 잘 보았습니다.
이효리님께서 ＂좋은 사람을 찾으려 하니 없었는데, 좋은 사람이 되려고 노력하니 좋은 사람이 오더라.＂ 라는 말씀을 하셨죠.
좋은 사람이 되어야겠다고 생각했던 제가 지금 좋은 사람이 되기 위해 노력하고 있는지 되돌아보는 계기가 되었네요 ㅎㅎ
그리고 향로님 같은 좋은 생각을 하는 분 밑에서 많은 걸 배워보고 싶다는 목표를 가지게 되었습니다.
비록 지금은 개발을 늦게 시작해서 많이 부족하지만 빠르게 실력을 키워서 기회가 닿을 수 있도록 노력해야겠습니다. 🚀

**Comment by JaydenLee1116** [comment]
매번 정말 좋은 글들 작성해주셔서 감사합니다. :)
